### PR TITLE
Fixed being unable to use survey tools, caused by server reboots

### LINKED ISSUE #1170: [Ressources] Error when sampling ressources
Go artisan
Find a good spot
Survey
Take a sample

error pops up
image

### PR DESCRIPTION
**Pull request by madsboddum**
Slight refactor of the code, to avoid having GalacticResource reference ServerData

Should fix #1170 and #1169.

### CODE REVIEW

File: src/main/java/com/projectswg/holocore/resources/gameplay/crafting/survey/SampleLoopSession.java [hunk 1]
```diff
@@ -311,7 +314,8 @@ private void sendSampleEffects() {
 	}
 	
 	private String getMusicFile() {
-		RawResource rawResource = resource.getRawResource();
+		RawResource rawResource = ServerData.INSTANCE.getRawResources()
+				.getResource(resource.getRawResourceId());
```
Comment (Josh-Larson) [inline src/main/java/com/projectswg/holocore/resources/gameplay/crafting/survey/SampleLoopSession.java:318]: Oh no, the multi-line chain strikes again

Comment (madsboddum) [reply src/main/java/com/projectswg/holocore/resources/gameplay/crafting/survey/SampleLoopSession.java:318]: Haha, indeed. The new code style does seem to have fixed this case.
I just ran it on this file and pushed the new version.

File: src/main/java/com/projectswg/holocore/resources/gameplay/crafting/survey/SurveyHandler.java [hunk 2]
```diff
@@ -225,7 +226,8 @@ private boolean isAllowedToSurvey(Location location, SurveyToolResolution resolu
 	}
 	
 	private static String getMusicFile(GalacticResource resource) {
-		RawResource rawResource = resource.getRawResource();
+		RawResource rawResource = ServerData.INSTANCE.getRawResources()
+				.getResource(resource.getRawResourceId());
```
Comment (Josh-Larson) [inline src/main/java/com/projectswg/holocore/resources/gameplay/crafting/survey/SurveyHandler.java:230]: Got one more for ya

File: src/main/java/com/projectswg/holocore/services/gameplay/crafting/resource/ResourceService.java [hunk 3]
```diff
@@ -40,20 +40,18 @@
...
 		this.spawner = new GalacticResourceSpawner();
 		this.spawnerUpdater = new BasicScheduledThread("resource-spawn-updater", spawner::updateAllResources);
 	}
 	
 	@Override
 	public boolean initialize() {
-		container.loadResources();
-		for (RawResource rawResource : container.getResources()) {
+		for (RawResource rawResource : ServerData.INSTANCE.getRawResources()
+				.getResources()) {
```
Comment (Josh-Larson) [inline src/main/java/com/projectswg/holocore/services/gameplay/crafting/resource/ResourceService.java:54]: Two more

Comment (madsboddum) [reply src/main/java/com/projectswg/holocore/services/gameplay/crafting/resource/ResourceService.java]: Ran code formatting with the new code style in those two files and it seems to have fixed the line breaks, among other things.

**Comment by Undercova** [comment]
Working as intended

**Pull request merged**